docs/src/man/getting_started.md

```diff
@@ -699,36 +699,6 @@ julia> mean(df.A)
 2.5
 ```
 
-### Column-Wise Operations
```

Comment:
Why not show how to get the same result with `select`? Though this probably doesn't need a separate section heading as it can be seen as a complement to `describe`.

Reply:
added